bleak/backends/p4android/recipes/bleak/fix_setup.py

```diff
@@ -0,0 +1,45 @@
+from setuptools import find_packages, setup
+
+VERSION = "[VERSION]"  # Version will be filled in by the bleak recipe
+
+# Package meta-data.
+NAME = "bleak"
+DESCRIPTION = "Bluetooth Low Energy platform Agnostic Klient"
+URL = "https://github.com/hbldh/bleak"
+EMAIL = "henrik.blidh@nedomkull.com"
+AUTHOR = "Henrik Blidh"
+
+# Where the magic happens:
+setup(
+    name=NAME,
+    version=VERSION,
+    description=DESCRIPTION,
+    author=AUTHOR,
+    author_email=EMAIL,
+    url=URL,
+    packages=find_packages(exclude=("tests", "examples", "docs", "BleakUWPBridge")),
+    entry_points={"console_scripts": ["bleak-lescan=bleak:cli"]},
+    test_suite="tests",
+    include_package_data=True,
+    license="MIT",
+    classifiers=[
```

Comment:
Since this file is just the minimum needed to install bleak, it would be nice to leave out all of the non-essential parts, e.g. classifiers, entry_points, test_suite, url, author, description, ...

Reply:
So these are the only ones left?

setup(
    name=NAME,
    version=VERSION,
    packages=find_packages(exclude=("tests", "examples", "docs", "BleakUWPBridge")),
)

Not sure about packages, aldough not strictly necessary, it might give problems/be less clean without it.

Reply:
Yes, I think those are the required ones. the packages can probably be changed to `find_packages(include=["bleak*"])` rather than the long list of excludes.

Reply:
`BleakUWPBridge` was removed years ago, so is no longer applicable

Reply:
I have changed in the latest commit